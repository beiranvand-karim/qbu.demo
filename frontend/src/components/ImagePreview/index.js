import React from "react"

const ImagePreview = ({image}) => {
   if (image) {
      return <img src={image} alt=""/>
   }
   return null
};

export default ImagePreview