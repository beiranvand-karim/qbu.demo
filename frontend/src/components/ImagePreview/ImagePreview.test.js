/* global describe, it, expect */
import React from "react"
import ImagePreview from "./"
import {shallow} from "enzyme"
import renderer from 'react-test-renderer'
import {MemoryRouter} from "react-router-dom"

describe('<ImagePreview />', () => {
   it('should renderer without crashing', () => {
      shallow(<ImagePreview />)
   });

   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><ImagePreview /></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});