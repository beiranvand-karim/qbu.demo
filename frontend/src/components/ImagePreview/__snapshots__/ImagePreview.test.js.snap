// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImagePreview /> should matches the snapshot 1`] = `null`;
